sample_size/scripts/utils.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict
```

Comment:
Wonder if this file can be me named better than a generic `utils.py`. This seems to be doing bulk of the input processing right?
Also, have we considered using argparse for input processing? Which is typically standard for CLI tools..

Reply:
will update to input_utils. I'll add a comment to consider using click.